code/modules/medical/cloner_defects.dm

```diff
@@ -450,24 +499,61 @@ ABSTRACT_TYPE(/datum/cloner_defect/organ_damage)
 
 	on_add()
 		. = ..()
-		if (data["trait_id"])
+		if (data["trait_id"] && !src.owner.traitHolder?.hasTrait(data["trait_id"]))
 			src.owner.traitHolder.addTrait(data["trait_id"])
-		else
+		else if (src.owner.bioHolder.HasEffect(data["bioeffect_id"]))
 			src.owner.bioHolder.AddEffect(data["bioeffect_id"])
 			var/datum/bioEffect/effect = src.owner.bioHolder.GetEffect(data["bioeffect_id"]) // this suuuucks
 			effect.curable_by_mutadone = FALSE
 
+	on_remove()
+		if (data["trait_id"] && !src.owner.traitHolder?.hasTrait(data["trait_id"]))
+			src.owner.traitHolder.removeTrait(data["trait_id"])
+		else if (src.owner.bioHolder.HasEffect(data["bioeffect_id"]))
+			src.owner.bioHolder.RemoveEffect(data["bioeffect_id"])
+		..()
+
+/datum/cloner_defect/biotrait/plasmalungs
+	name = "Plasma Lung Infection"
+	desc = "Subject's lungs have become infected and are now only capable of breathing plasma."
+	severity = CLONER_DEFECT_SEVERITY_MAJOR
+	weight = 50
+	effect_type_pool = list(/datum/trait/plasmalungs)
```

Comment:
Due to the nature of how this is typically contracted, I don't think it makes much sense for this to be a generic abnormality. Unless it is rare.

Reply:
I had a similar thought. Perfectly okay with making it rare, or cutting it entirely otherwise.

Reply:
Either one.  This requires a lung transplant or "immediate" coordination with Science or Engineer doesn't it?

Reply:
It's a bit more complicated because of the way the trait is coded, replacing the lungs does not fix it, which imho is an issue with the trait itself and very confusing design. Intermediate solution is to supply the patient with salbutamol until plasma can be ordered.

Because replacing the lungs does not fix it, I would actually argue this should be culled, at least not as long as the trait remains that way. Too unintuitive.